examples/experiments/tracker_dealer_doublesided.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+"""
+======================
+Tracker Dealer Example

```

Comment:
name should be different from the other one, perhaps "Adaptive tracking from above and below" (and don't use Title Case)